src/index.js

```diff
@@ -186,31 +147,41 @@ const ATD_DocklessMap = (function() {
       });
     });
 
-    function postCellTripCount(feature, divId = "dataPane") {
-      var trip_percent =
+    function renderCellTripCount(feature, divId = "js-data-pane") {
+      const trip_percent =
         feature.properties.current_count / docklessMap.total_trips;
-
-      if (docklessMap.flow == "origin") {
-        var text =
-          docklessMap.formatKs(feature.properties.current_count) +
-          " (" +
-          docklessMap.formatPct(trip_percent) +
-          ") trips originated in the clicked cell.";
-      } else if (docklessMap.flow == "destination") {
-        var text =
-          docklessMap.formatKs(feature.properties.current_count) +
-          " (" +
-          docklessMap.formatPct(trip_percent) +
-          ") trips terminated in the clicked cell.";
+      let text;
```

Comment:
a lot of the changes I made in here are just syntax that es6 gives us. string interpolation, let and const instead of var, triple equals for equality without type coercion, etc.